### ISSUE TITLE
Feature Request: Adjust Sync of Lines From Selection Onwards by Clicking on the Audio Window

### ISSUE THREAD

**Issue by MadokaAyukawa** [wangqr/Aegisub#68]
I saw this in the Subtitle Edit help page. 

`Mouse single click+Ctrl+Shift will set of start selected line + offset all lines after selected line (keeping durations)`

I re-sync subtitles all the time, but it's a pain when you're adjusting for CM breaks. I realize that I can do it with the Shift Times... feature, but almost always I can tell where I want the subs to start by looking at the audio window. It would be great to be able to select the first line you want to adjust, hold ctrl+shift(or alt)+left click and have Aegisub move everything from that line to the end of file en bloc to the new time while keeping durations. 

I currently can do this with Alt+L-click/drag, but I have to select that line to the end of file every time, so there's a ton of scrolling on the subtitle grid. Or alternative to setting the start position as above, make Alt+Ctrl+L-click/drag affect the selected line inclusive to the end of the file without having to manually select those lines first.

**Comment by wangqr** [comment]
As a side note, you can use shift+end to select all lines until last one in subtitle grid. Scrolling back to find the starting point is still needed afterwards though.

**Comment by MadokaAyukawa** [comment]
Yes, that's what I do now. The problem is that Aegisub then sets the last line in the file as the active line, and then the audio window is scrolled to the end of the file. I then have to scroll up to the top of the selection, set the first line as the active line to get the audio window back to the place/line I need, then do Alt+L-click/drag.

Ideally the second option above would be ideal (Alt+Ctrl+L-click/drag). I would even be happy if Aegisub would keep the first line as the active line when using Shift+End so that I can directly go to the audio window to adjust the timing.
